commands_to_load/Reminders.py

```diff
@@ -89,12 +95,15 @@ def __init__(self, bot):
 				author = ctx.author.nick or ctx.author.name
 				if reminders != '':
 					logger.info("[Reminders showreminders()] sent off the list of reminders to "+str(ctx.message.author))
-					await ctx.send("Here are your reminders "+ author+"\n```"+"MessageID\t\t\tReminder\n"+reminders+"```")
+					eObj = embed(title="Here are you reminders " + author, author=settings.BOT_NAME, avatar=settings.BOT_AVATAR, content=[["MessageID\t\t\tReminder", reminders]])
```

Comment:
can this be fixed so that the "Reminders" heading actually appears above the reminder and not over half of the messageID?

Reply:
ill work on it tonight

Reply:
@modernNeo done. doth u approve